src/communication.ts

```diff
@@ -25,11 +48,18 @@ class Communication {
     return Communication.instance
   }
 
-  broadcast (code: MESSAGE_CODE, payload?: Record<string, any>) {
+  broadcast (code: MessageCodesEnum.I_AGREEMENT_NEW, payload: AgreementInfoPayload): void
+  broadcast (code: MessageCodesEnum.I_AGREEMENT_STOPPED, payload: AgreementInfoPayload): void
+  broadcast (code: MessageCodesEnum.I_AGREEMENT_EXPIRED, payload: AgreementInfoPayload): void
+  broadcast (code: MessageCodesEnum.I_HASH_PINNED, payload: HashInfoPayload): void
+  broadcast (code: MessageCodesEnum.I_HASH_START, payload: HashInfoPayload): void
+  broadcast (code: MessageCodesEnum.W_HASH_RETRY, payload: RetryPayload): void
+  broadcast (code: MessageCodesEnum.E_HASH_NOT_FOUND, payload: HashInfoPayload): void
+  broadcast (code: MessageCodesEnum.E_AGREEMENT_SIZE_LIMIT_EXCEEDED, payload: AgreementSizeExceededPayload): void
+  broadcast (code: MessageCodesEnum, payload?: Record<string, any>): void
+  broadcast (code: MessageCodesEnum, payload?: Record<string, any>): void {
     logger.error(`NOT IMPLEMENTED - broadcasting message with code ${code}`, payload)
   }
 }
 
-const instance = Communication.getInstance()
-
-export default instance
+export const channel = Communication.getInstance()
```

Comment:
Why did you change this from default export?

Reply:
https://github.com/AuHau/tasegir/issues/77

For example it can be subbed for tests, that I forgot to write 😅

Reply:
I found the way of stubbing default import. Check migration PR's test's